packages/yourloops/lib/utils.ts

```diff
@@ -152,3 +153,16 @@ export function fixYLP878Settings(settings: Settings | undefined | null): Settin
   }
   return newSettings;
 }
+
+/**
+ * Set the page title
+ * @param prefix The text prefix
+ * @param metricsTitle The text for the metrics, to keeps this page title anonymous
+ */
+export function setPageTitle(prefix: string, metricsTitle?: string): void {
```

Comment:
what about having the prefix optional, so that we don't make those weird calls `setPageTitle("")` and we can simply call `setPageTitle()` ?

```suggestion
export function setPageTitle(prefix?: string, metricsTitle?: string): void {
```

Reply:
Good point !
Reason: I didn't think hard enough I guess.